agents1/Message.py

```diff
@@ -0,0 +1,167 @@
+import enum
+import json
+import re
+
+from matrx.messages.message import Message
+
+
+class MessageType(enum.Enum):
```

Comment:
This should be a new class, otherwise the `MessageType` defined in our agent class would not equal this one